modules/config/main.tf

```diff
@@ -25,3 +25,21 @@ resource "aws_ssm_parameter" "graphdb_lb_dns_name" {
   type        = "String"
   value       = var.graphdb_lb_dns_name
 }
+
+resource "aws_ssm_parameter" "graphdb_properties" {
+  count = var.graphdb_properties_path != null ? 1 : 0
+
+  name        = "/${var.resource_name_prefix}/graphdb/graphdb_properties"
+  description = "Additional properties to append to graphdb.properties file."
+  type        = "SecureString"
+  value       = filebase64(var.graphdb_properties_path)
+}
+
+resource "aws_ssm_parameter" "gdb_java_opts" {
```

Comment:
Use consistent naming for the resources ?

Reply:
fixed